src/app/onboarding/components/glam/select-archive-type-screen/select-archive-type-screen.component.ts

```diff
@@ -12,29 +12,33 @@ import {
   templateUrl: './select-archive-type-screen.component.html',
   styleUrl: './select-archive-type-screen.component.scss',
 })
-export class SelectArchiveTypeScreenComponent {
+export class SelectArchiveTypeScreenComponent implements OnInit {
   selectedValue = '';
-  buttonText = '';
+  buttonText = 'a Personal';
   public headerText: string = '';
-  public tag: string = '';
+  @Input() tag: string = '';
   public type: string = '';
 
-  @Output() navigationEmitter = new EventEmitter<string>();
   @Output() submitEmitter = new EventEmitter<Record<string, string>>();
 
-  public navigate(screen) {
-    if (screen === 'start') {
-      this.navigationEmitter.emit(screen);
-    } else {
-      this.submitEmitter.emit({
-        screen,
-        type: this.type,
-        tag: this.tag,
-        headerText: this.headerText,
-      });
+  ngOnInit(): void {
+    if (this.tag) {
+      this.buttonText = generateElementText(
+        this.tag,
+        archiveOptionsWithArticle,
+      );
     }
```

Comment:
Codecov is pointing out that this is missing coverage. It would be nice to write a test that renders the component with the `tag` input provided.